install_scripts/install_sindri.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+# Install sindri and related software
+
+if [[ $# -ne 1 ]]; then
+    echo "Pass the sensor number"
+    exit 1
+fi
+
+name="hamma"$1  # This will be used for the server push location
+
+# We need to modify a file (for now). Set up the variables for that
+LEKTOR_FILE="/home/pi/dev/sindri/src/sindri/website/mjolnir-website/mjolnir-website.lektorproject"
+LEKTOR_KEY="target = rsync://pi@hamma.dev/var/www/hamma.dev/public_html/"
```

Comment:
As you imply here, Sindri 0.3.0a1 and above no longer requires this (though may as well leave it until a stable-enough version is actually deployed to the Pis...though as of now, that version will be pulled in already with these scripts, since they pull the latest `master` dev version rather than a specific tag or the latest stable.